dist/ngtweet.js

```diff
@@ -125,16 +126,17 @@ function TwitterWidget(ngTweetLogger, TwitterWidgetFactory) {
         transclude: true,
         scope: {
             twitterWidgetId: '=',
-            twitterWidgetOptions: '='
+            twitterWidgetOptions: '@'
```

Comment:
Since you're accessing this attribute via `attrs`, it isn't necessary to declare it in this block.